src/index.ts

```diff
@@ -1,4 +1,6 @@
-import { type LegacyException, type LegacyResult, render } from 'sass';
+import { type LegacyException, type LegacyResult } from 'sass-embedded';
+
+const { render } = require('sass-embedded');
```

Comment:
Do we have to use `require`?
```suggestion
import { type LegacyException, type LegacyResult, render } from 'sass-embedded';
```

Reply:
I did not get it to work with `import`, but this was after a few hours of fighting obtuse issues with `sass`, so it is possible I missed it.

Reply:
I had a ton of error during the build, but I figured out that I needed to declare this external dependency (now that sass is not inlined like before) and then this works.

I am uncertain wether this actually turns `sass-embedded` into a peer dependency, but I think this would be advantageous if the case, as it should allow individual projects to update to newer versions without this plugin needing a bump (until 2.0 that is, when the legacy API goes away)